Give possibility to use external mask

LSchuepbach opened the pull request:
I've unfortunately had bad results from both masking possibilities internal to FIACH. I propose to give the possibility to a user to provide their own mask.

LSchuepbach committed:
Give possibility to use external mask

tierneytim reviewed (changes requested):
Hi, I think adding functionality for an external mask is a great idea. However, there should probably be a check for external masks when outputting data(line 266(old) 282(new))

tierneytim commented on R/fiach.R:
```diff
@@ -263,7 +279,7 @@ fiach <-function(input,t,tr,rp=NULL,maxgap=1,freq=128,nMads = 1.96,defaultMask=T
   scrubbed.brain[scrub.inds]<-brain.meds[scrub.inds[,2]]
   output.brain.mat<-hp.mat                                                                    
-  #output.brain.mat[,which(mask.mat==FALSE)]<-0                                           
+  output.brain.mat[,which(mask.mat==FALSE)]<-0                                           
```
I think There should probably be a check here to see if the user has used an external mask before zeroing out data.

LSchuepbach committed:
Scrub outside mask only if external mask is provided

You are right @tierneytim. Fortunately, I had at line 106 a variable I tought I would probably use somewhere.

LSchuepbach commented:
Dear Tim, I've made the requested changes. And added it to the branch. Is there something else you'd like me to improve ?
